frontend/src/metabase/query_builder/containers/test-utils.tsx

```diff
@@ -246,7 +246,6 @@ export const setup = async ({
     setupModelIndexEndpoints(card.id, []);
   }
 
-  // this workaround can be removed when metabase#34523 is fixed
```

Comment:
_This is totally unrelated, just found during triage and I didn't want to have a separate PR for it._

#34523 was fixed, but this comment was inaccurate - we still need to mock `/model-index` endpoint which is used after a model is created.